src/backends/plonky2/mainpod/mod.rs

```diff
@@ -293,9 +293,7 @@ impl PodProver for Prover {
             .signed_pods
             .iter()
             .map(|p| {
-                let p = p
-                    .as_any()
-                    .downcast_ref::<SignedPod>()
+                let p = (*p as &dyn Any).downcast_ref::<SignedPod>()
                     .expect("type SignedPod");
```

Comment:
```suggestion
                let p = (*p as &dyn Any)
                    .downcast_ref::<SignedPod>()
                    .expect("type SignedPod");
```
cf. `cargo fmt --check`.